Detectors/FIT/FT0/simulation/include/FT0Simulation/Detector.h

```diff
@@ -135,6 +135,96 @@ class Detector : public o2::base::DetImpl<Detector>
   /// Define the sensitive volumes of the geometry
   void defineSensitiveVolumes();
 
+  // Define the aluminium frame for the detector
+  TGeoVolume* ConstructFrameGeometry();
+  std::string frame1CompositeShapeBoolean();
+  std::string frame2CompositeShapeBoolean();
+  std::string frameCompositeShapeBoolean();
+  std::string plateGroupCompositeShapeBoolean();
+  std::string opticalFiberPlateCompositeShapeBoolean1();
+  std::string opticalFiberPlateCompositeShapeBoolean2();
+  std::string pmtCornerCompositeShapeBoolean();
+  std::string pmtCompositeShapeBoolean();
+  std::string plateBoxCompositeShapeBoolean();
+  void defineTransformations();
+  void defineQuartzRadiatorTransformations();
+  void definePMTTransformations();
+  void definePlateTransformations();
+  void defineFrameTransformations();
+
+  // define some error to avoid overlaps
+  Float_t eps = 0.025;
+
+  // frame 1 has a longer side horizontal
```

Comment:
Currently it is quite hard to understand what is rect1, rect2, etc. Perhaps if you could make a comment with simple ASCII drawing it would be helpful. As an example see: https://github.com/mslupeck/AliceO2/blob/dev/Detectors/FIT/FV0/base/src/Geometry.cxx#L533